src/controllers/controllermanager.cpp

```diff
@@ -364,6 +385,10 @@ void ControllerManager::closeController(Controller* pController) {
     // Update configuration to reflect controller is disabled.
     m_pConfig->setValue(ConfigKey(
         "[Controller]", presetFilenameFromName(pController->getName())), 0);
+
+    if (dynamic_cast<void*>(pController) == dynamic_cast<void*>(getKeyboardController())) {
```

Comment:
This feels a bit hacky. Couldn't Mixxx not diretcly connect to the keyboard controller?